<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrlCoderDecoderWidget</class>
 <widget class="QWidget" name="UrlCoderDecoderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="spacePlusButton">
         <property name="text">
          <string>Space as +</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="space20Button">
         <property name="text">
          <string>Space as %20</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Decoded</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openDecodedButton">
           <property name="toolTip">
            <string>Open</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyDecodedButton">
           <property name="toolTip">
            <string>Copy</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pasteDecodedButton">
           <property name="toolTip">
            <string>Paste</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditPaste"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearDecodedButton">
           <property name="toolTip">
            <string>Clear</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditClear"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CodeEditor" name="decoded"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Encoded</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openEncodedButton">
           <property name="toolTip">
            <string>Open</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyEncodedButton">
           <property name="toolTip">
            <string>Copy</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pasteEncodedButton">
           <property name="toolTip">
            <string>Paste</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditPaste"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearEncodedButton">
           <property name="toolTip">
            <string>Clear</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditClear"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CodeEditor" name="encoded"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
